bliss/encoder.py

```diff
@@ -324,6 +370,10 @@ def test_step(self, batch, batch_idx):
 
     def predict_step(self, batch, batch_idx, dataloader_idx=0):
         """Pytorch lightning method."""
+        # NOTE: Debugging predict issue in test_simulate.py - extra dimension
```

Comment:
Can we fix this issue in `test_simulate.py` rather than introducing an extra conditional block in the encoder?